build-tools-internal/src/main/resources/changelog-schema.json

```diff
@@ -24,6 +24,7 @@
             "Audit",
             "Authentication",
             "Authorization",
+            "AutoConfiguration",
```

Comment:
Have we checked with the ES Docs team whether this is the heading they want to use in the release notes?

Reply:
I have not, because it's all a bit unclear to me (I just needed a way of categorising GitHub issues).
Is there somewhere I can get an explanation of how it all hangs together?

Reply:
I ping you with some docs, let me know if anything's unclear after after. Fundamentally the `area` in the change is how the change will appear in the release notes, and we have automation that attempts to derive / map the team label in GitHub to the release notes area. I mentioned the Docs team because they may prefer e.g. `Auto-configuration`. We could also just tweak the GitHub label 😁